src/infra/queue/adapter/bull.js

```diff
@@ -35,7 +35,9 @@ export default class BullAdapter {
     const options = {
       priority,
       delay,
-      timeout: ttl
+      timeout: ttl,
+      attempts: 3,
```

Comment:
for the hull-intercom failed jobs because of intercom rate limits we should explicitely reschedule those those jobs instead of letting them fail..., @michaloo what do you think ?